rust/lance-encoding/src/decoder.rs

```diff
@@ -1858,6 +1876,7 @@ fn create_scheduler_decoder(
             config.io.clone(),
             config.cache,
             &filter,
+            config.cache_repetition_index,
```

Comment:
Same comment as elsewhere.  Maybe it is easier at this point to just pass `config` to `DecodeBatchScheduler::try_new`.